dev/create-release/release-build.sh

```diff
@@ -392,6 +375,7 @@ if [[ "$1" == "publish-release" ]]; then
   find . -type f |grep -v \.jar |grep -v \.pom | xargs rm
 
   echo "Creating hash and signature files"
+  # this must have .asc, .md5 and .sha1 - it really doesn't like anything else there
```

Comment:
sha1, not sha512?

Reply:
it rejects it when there are "extra files" of the sha512 extension.